src/pages/header/header.component.html

```diff
@@ -5,6 +5,6 @@ <h3>My forms</h3>
 </div>
   <div class="header-link">
     <h4>User</h4>
-    <a href="#" class="header-link_logout">Log out</a>
+    <a routerLink="/login" class="header-link_logout">Log out</a>
```

Comment:
you should not only redirect user to login page, but also remove token from browser storage(add logout method to user service and redirect to login after token removed)